### PR TITLE
Update README, add LICENSE and CONTRIBUTING

### PR DESCRIPTION
**Pull request by markbackman**

### CODE REVIEW

**Comment by markbackman** [comment]
@filipi87 Feel free to modify the README to be more accurate.

**Comment by markbackman** [comment]
Also, please review these docs: https://docs.pipecat.ai/client/react-native/introduction. You can submit a PR at: https://github.com/pipecat-ai/docs

**Comment by filipi87** [comment]
> Also, please review these docs: https://docs.pipecat.ai/client/react-native/introduction. You can submit a PR at: https://github.com/pipecat-ai/docs

Yep, I will take a look on this one next.

File: README.md [hunk 1]
```diff
@@ -1,8 +1,17 @@
...
+<h1><div align="center">
+ <img alt="pipecat client react native" width="500px" height="auto" src="https://raw.githubusercontent.com/pipecat-ai/pipecat-client-react-native-daily-transport/main/pipecat-react-native.png">
+</div></h1>
 
-[RTVI-AI](https://github.com/rtvi-ai/) is an open standard for Real-Time Voice [and Video] Inference.
+[![Docs](https://img.shields.io/badge/documentation-blue)](https://docs.pipecat.ai/client/introduction)
+![NPM Version](https://img.shields.io/npm/v/@pipecat-ai/react-native-daily-transport)
 
-This library exports a VoiceClient that has the [Daily](https://www.daily.co/) transport associated.
+The official React Native client SDK for [Pipecat](https://github.com/pipecat-ai/pipecat), an open source Python framework for building voice and multimodal AI applications.
```
Comment (filipi87) [inline README.md:8]: The only thing I’m not sure about is whether we could improve this or not. Currently, we don’t have an SDK specifically for React Native; instead, we use the same plain JavaScript SDK, @pipecat-client/client-js.

The difference in this case lies in the Daily transport. We have one Daily transport for the web and this one for React Native.

Comment (filipi87) [reply README.md:8]: What do you think @markbackman ?

Comment (markbackman) [reply README.md:8]: Ahh, yes. Maybe we shy away from that wording then. Good point.

Comment (markbackman) [reply README.md]: I just made a change.

**Review by filipi87 — APPROVED** [review]
Looks good. Thanks for doing this @markbackman . 🚀

**Pull request merged**